### ISSUE TITLE
Paginators' nextToken should be undefined during the first request

### ISSUE THREAD

**Issue by fcoelho** [aws/aws-sdk-js-v3#1517]
**Describe the bug**
When using paginators, the first request should not include a `nextToken`. Paginator functions are currently sending an empty string from here: https://github.com/aws/aws-sdk-js-v3/blob/15fb1ea26a590efbf0122db468e33d4b3b7c9288/clients/client-cloudwatch-logs/pagination/DescribeLogGroupsPaginator.ts#L32

**SDK version number**

1.0.0-gamma.8

**Is the issue in the browser/Node.js/ReactNative?**

Node.js

**Details of the browser/Node.js/ReactNative version**

```
$ node -v
v12.18.3
```

**To Reproduce (observed behavior)**

```ts
import {
  CloudWatchLogsClient,
  describeLogGroupsPaginate,
} from "@aws-sdk/client-cloudwatch-logs";

(async () => {
  const paginator = describeLogGroupsPaginate(
    { client: new CloudWatchLogsClient({}) },
    {},
    {}
  );

  try {
    for await (const page of paginator) {
      console.log(page);
    }
  } catch (exc) {
    console.log(exc);
  }
})();
```

```bash
$ npm start

> cloudwatch-logs-pagination@1.0.0 start /tmp/cloudwatch-logs-test
> ts-node main.ts

InvalidParameterException: 1 validation error detected: Value '' at 'nextToken' failed to satisfy constraint: Member must have length greater than or equal to 1
    at deserializeAws_json1_1DescribeLogGroupsCommandError (/tmp/cloudwatch-logs-test/node_modules/@aws-sdk/client-cloudwatch-logs/protocols/Aws_json1_1.ts:1930:39)
    at processTicksAndRejections (internal/process/task_queues.js:97:5) {
  '$fault': 'client',
  '$metadata': {
    httpStatusCode: 400,
    httpHeaders: {
      'x-amzn-requestid': 'xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx',
      'content-type': 'application/x-amz-json-1.1',
      'content-length': '184',
      date: 'Fri, 11 Sep 2020 10:10:42 GMT',
      connection: 'close'
    },
    requestId: 'xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx',
    attempts: 1,
    totalRetryDelay: 0
  },
  __type: 'InvalidParameterException'
}
```

```json
# tsconfig.json
{
  "compilerOptions": {
    "module": "commonjs",
    "moduleResolution": "node",
    "noUnusedLocals": true,
    "noUnusedParameters": true,
    "sourceMap": true,
    "target": "es6",
    "esModuleInterop": true,
    "resolveJsonModule": true,
    "downlevelIteration": true,
    "preserveSymlinks": true,
    "types": [
      "node"
    ]
  }
}
```

```json
# package.json
{
  "name": "cloudwatch-logs-pagination",
  "version": "1.0.0",
  "dependencies": {
    "@aws-sdk/client-cloudwatch-logs": "^1.0.0-gamma.8",
    "typescript": "^4.0.2",
    "ts-node": "^9.0.0"
  },
  "scripts": {
    "start": "ts-node main.ts"
  }
}

```

**Expected behavior**

Retrieving the actual log groups

**Comment by alexforsyth** [comment]
Sweet! Thanks for opening this - I'll take a look today

**Comment by trivikr** [comment]
This issue was fixed in https://github.com/aws/aws-sdk-js-v3/pull/1597, and was published in **v1.0.0-rc.2**
